src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisWithActionsTest.java

```diff
@@ -839,7 +839,7 @@ void testLoadAction() {
     }
 
     private void testGeneratorAction(boolean dc, LoadFlowParameters.BalanceType balanceType, double deltaG1, double deltaG2,
```

Comment:
I don't understand why you have changed the method signature and the tolerances. What "We need a larger delta power assert since we distribute some power in slack distribution steps" means?

Reply:
I guess that the second question is solved with inputs from @vidaldid-rte.

I changed the method signature because it seems not very satisfying to have the slack active power mismatch that depends on the test. It seems better to use the NewtonRaphsonStoppingCriteriaType.PER_EQUATION_TYPE_CRITERIA criterion as @vidaldid-rte suggested.